Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml

```diff
@@ -0,0 +1,2742 @@
+    scriptarguments:
+      index:
+        complex:
+          root: inputs.ElasticIndex
+      query:
+        simple: (process.executable.text:"*\\w3wp.exe" AND file.path.text:"*.aspx*")
+      timestamp_field:
+        complex:
+          root: inputs.ElasticEarliestTime
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 330,
+          "y": 1730
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: 06b72db6-e3b8-4cd2-8eb2-0febbed6971e
+    type: regular
+    task:
+      id: 06b72db6-e3b8-4cd2-8eb2-0febbed6971e
+      version: -1
+      name: Collect IoCs from Huntress
+      description: This script will extract indicators from given HTML and will handle bad top-level domains to avoid false positives caused by file extensions.
+      scriptName: ParseHTMLIndicators
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      url:
+        simple: https://www.huntress.com/blog/moveit-transfer-critical-vulnerability-rapid-response
+    reputationcalc: 2
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 330,
+          "y": 430
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: 521f06be-e2d7-41bc-8f07-276fa1464308
+    type: regular
+    task:
+      id: 521f06be-e2d7-41bc-8f07-276fa1464308
+      version: -1
+      name: Tag IP Indicators
+      description: Create indicators to the Threat Intel database only if they are not registered. When using the script with many indicators, or when the Threat Intel Management database is highly populated, this script may have low performance issue.
+      scriptName: CreateNewIndicatorsOnly
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      indicator_values:
+        complex:
+          root: IP
+          accessor: Address
+      tags:
+        simple: Outlook, 0-day, Microsoft
+      type:
+        simple: IP
+    reputationcalc: 2
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1190,
+          "y": 750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: 9d3492f4-1d99-444a-8ed1-d8eac62cb083
+    type: playbook
+    task:
+      id: 9d3492f4-1d99-444a-8ed1-d8eac62cb083
+      version: -1
+      name: QRadarFullSearch
```

Comment:
need to change the names of Qradar :)  (few tasks)

Reply:
Done